dissect/target/helpers/localeutil.py

```diff
@@ -0,0 +1,36 @@
+import defusedxml.ElementTree as ET
+from locale import normalize
+
+
+windows_zones = {
+    child.attrib["other"]: child.attrib["type"]
+    for child in ET.parse("./dissect/target/helpers/data/windowsZones.xml").findall(
```

Comment:
Can you use [importlib.resources](https://docs.python.org/3/library/importlib.resources.html) instead? [Example usage in dissect.etl](https://github.com/fox-it/dissect.etl/blob/main/dissect/etl/manifest.py#L294-L309)

May require adding `package_data` to `setup.py`. [Example in dissect.etl again](https://github.com/fox-it/dissect.etl/blob/main/setup.py)

The reason I'm asking is because I don't think this current method is guaranteed to work in all different possible ways you can install or use Python package.

Reply:
Where would you want us to define `get_resource_string` and `_get_resource_path`? Inside `localeutil.py` or some other generic util file?

Reply:
Added in [cba0ede](https://github.com/fox-it/dissect.target/pull/150/commits/cba0edec2706c65cd9e982c7155f61d67ab88ab4).

Reply:
`localeutil.py` is fine for now, until we start needing that elsewhere.